src/DynamoCore/Configuration/GroupStyleItem.cs

```diff
@@ -14,10 +15,10 @@ public class GroupStyleItem: StyleItem
         /// </summary>
         public static ObservableCollection<GroupStyleItem> DefaultGroupStyleItems =
             new ObservableCollection<GroupStyleItem>() {
-                new GroupStyleItem() { Name = Resources.GroupStyleDefaultActions, HexColorString = Resources.GroupStyleDefaultActionsColor, FontSize = 36, IsDefault = true },
-                new GroupStyleItem() { Name = Resources.GroupStyleDefaultInputs, HexColorString = Resources.GroupStyleDefaultInputsColor, FontSize = 36, IsDefault = true },
-                new GroupStyleItem() { Name = Resources.GroupStyleDefaultOutputs, HexColorString = Resources.GroupStyleDefaultOutputsColor, FontSize = 36, IsDefault = true },
-                new GroupStyleItem() { Name = Resources.GroupStyleDefaultReview, HexColorString = Resources.GroupStyleDefaultReviewColor, FontSize = 36 ,IsDefault = true }
+                new GroupStyleItem() { Name = Resources.GroupStyleDefaultActions, HexColorString = Resources.GroupStyleDefaultActionsColor, FontSize = 36, IsDefault = true, GroupStyleId = new Guid("4d68be4a-a04d-4945-9dd5-cdf61079d790") },
```

Comment:
You can use the guid method to create a static one instead of hardcoding it here.

Reply:
actually this is intentional because those styles are by default, otherwise we will have different Ids per user / session

Reply:
I agree these ids should be const